integration-test/README.md

```diff
@@ -0,0 +1,59 @@
+End to end storm integration tests
+==================================
+
+Running tests end-to-end
+------------------------
+Assumption:
+A single version of storm binary zip such as `storm-dist/binary/target/apache-storm-2.0.0-SNAPSHOT.zip` is present
+The following command will bring up a vagrant cluster.
+```sh
+cd integration-test/config
+vagrant up
+```
+This automatically will run `integration-test/run-it.sh`.
+This brings up a vagrant machine, with storm and zookeeper daemons.
+And runs all the tests against it.
+
+Running tests for development & debugging
+=========================================
+```vagrant up``` command is steup as a complete auto-pilot.
+Following describes how we can run individual tests against this vagrant cluster or any other cluster.
+
+Configs for running
+-------------------
+The supplied configuration will run tests against vagrant setup. However, it can be changed to use a different cluster.
+Change `integration-test/src/test/resources/storm.yaml` as necessary.
+
+Running all tests manually
+--------------------------
+To run all tests:
+```sh
+mvn clean package -DskipTests && mvn test

```

Comment:
`mvn test` will make these integration tests run in the maven `test` phase. I believe it would be more appropriate to have these integration tests run in the maven `integration-test`  or `verify` phase.

Please refer [here](https://github.com/apache/storm/edit/master/DEVELOPER.md#L122) for relevant info.